### ISSUE TITLE
WooCommerce spanish language

### ISSUE THREAD

**Issue by andres-pinilla** [btcpayserver/btcpayserver#94]
The WooCommerce plugin is not translated into Spanish. The plugin does not have a .po to start the task.,

Andrés

**Comment by NicolasDorier** [comment]
I am unsure how to do that, I am not familiar with woocommerce too much. Can you do a PR to the woocommerce plugin to add this if you know how to do?

**Comment by andres-pinilla** [comment]
Hi Nicolas, thanks for your response.

In this link you can find the official WordPress documentation for localice
plugins.

-- 
*Andrés Pinilla*
Gerente *Compudemano*
Distribuidor Autorizado *Apple*
*Google* Cloud Partner

Carrera 15 # 13-35. Pereira, Colombia
Teléfono: 335-9494
Celular/WhatsApp: (316) 425-4777
Bogotá: 381-9922 - Medellín: 204-0707
Cali: 891-2999 - Barranquilla: 316-1300
www.compudemano.com


El mar. 31, 2018, a la(s) 9:46 a. m., Nicolas Dorier <
notifications@github.com> escribió:

I am unsure how to do that, I am not familiar with woocommerce too much.
Can you do a PR to the woocommerce plugin to add this if you know how to do?

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub
<https://github.com/btcpayserver/btcpayserver/issues/94#issuecomment-377697970>,
or mute the thread

**Comment by britttttk** [comment]
@andres-pinilla it is possible:
[https://www.webinterpret.com/us/blog/ecommerce-translation-woocommerce/](https://www.webinterpret.com/us/blog/ecommerce-translation-woocommerce/)
[https://translate.wordpress.org/projects/wp-plugins/woocommerce](https://translate.wordpress.org/projects/wp-plugins/woocommerce)

**Comment by britttttk** [comment]
@andres-pinilla do you still have this problem?

**Comment by andres-pinilla** [comment]
Yes @britttttk, with each new version of the plugin I must translate some words manually
